oath/src/main/java/com/yubico/yubikit/oath/CredentialData.java

```diff
@@ -59,7 +59,7 @@ public class CredentialData implements Serializable {
     /**
      * Parses an <a href="https://github.com/google/google-authenticator/wiki/Key-Uri-Format">otpauth:// URI</a>.
      * <p>
-     * Example URI: <pre>otpauth://totp/Example:alice@google.com?secret=JBSWY3DPEHPK3PXP&amp;issuer=Example</pre>
+     * Example URI: <pre>otpauth://totp/Example:alice@google.com?secret=OTP_SECRET&amp;issuer=Example</pre>
```

Comment:
This is an example, and as such should be a valid otpauth:// value. This change breaks that.